Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.23.2 as build
+FROM golang:1.24.0 as build
```

Comment:
Should we pin this to `1.24` so that we get point releases automatically, or is that too risky for the CLI?

Reply:
The other advantage of pinning with x.y.z is cache busting.